### PR TITLE
Chore: Remove NewFirstInquiry feature flag

### PR DESCRIPTION
**Pull request by Mitch-Henson**
The type of this PR is: **chore**

This PR resolves [NX-3093]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [x] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Removes old NewFirstInquiry feature flag - Mitchell

</details>

[NX-3093]: https://artsyproduct.atlassian.net/browse/NX-3093?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Removes old NewFirstInquiry feature flag - Mitchell)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against be2937fc18964e51397ea9df7b8d217896299ecf
</p>

File: src/app/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx [hunk 1]
```diff
@@ -94,12 +77,7 @@ export const CommercialButtons: React.FC<CommercialButtonProps> = ({
       <>
         <MakeOfferButtonFragmentContainer artwork={artwork} editionSetID={editionSetID ?? null} />
         {isInquireable && <Spacer my={0.5} />}
-        {isInquireable && !newFirstInquiry && (
-          <Button onPress={handleInquiry} size="large" variant="outline" block width={100} haptic>
-            {InquiryOptions.ContactGallery}
-          </Button>
-        )}
-        {isInquireable && newFirstInquiry && (
+        {isInquireable && (
```
Comment (lidimayra) [inline src/app/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx:80]: I guess we could get rid of L79 if we move `<Spacer my={0.5}/>` to L81, couldn't we?

Comment (Mitch-Henson) [reply src/app/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx:79]: We absolutely could! Fixed in this commit: https://github.com/artsy/eigen/pull/6419/commits/aa181e868688acc8f9808a2fd3429bd7b8dd8d6a

**Review by araujobarret — APPROVED** [review]
Good clean up, thanks for doing it 💯 
Gonna pull it tomorrow to test, then merge ✅

**Pull request merged**